### PR TITLE
Added user creation steps for oracle

### PR DESCRIPTION
**Pull request by ShivanshGahlot**
Added steps to create the oracle user `ybvoyager` on the docker instance.

### CODE REVIEW

File: migtests/scripts/oracle/create_oracle_user [hunk 1]
```diff
@@ -1,2 +1,6 @@
 #!/usr/bin/env bash
+source migtests/scripts/postgresql/env.sh
```
Comment (priyanshi-yb) [inline migtests/scripts/oracle/create_oracle_user:2]: Change it to
```
source migtests/scripts/oracle/env.sh
```

**Review by priyanshi-yb — APPROVED** [review]
LGTM

File: migtests/scripts/oracle/create_oracle_user [hunk 2]
```diff
@@ -1,2 +1,9 @@
 #!/usr/bin/env bash
+source migtests/scripts/oracle/env.sh
```
Comment (amit-yb) [inline migtests/scripts/oracle/create_oracle_user:2]: Just add a comment at the top stating the purpose of the script i.e. to enable jenkins pipeline to create an oracle user. And relevant links to the code will also help.

**Review by amit-yb — APPROVED** [review]
LGTM with a minor suggestion.

**Pull request merged**